.github/workflows/release_and_publish.yml

```diff
@@ -15,9 +15,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.3.4
+    - uses: actions/checkout@v2
     - name: Set up Python
-      uses: actions/setup-python@v2.1.4
+      uses: actions/setup-python@v2
```

Comment:
@martinfleis I'm wondering if this can be swapped out for `conda-incubator/setup-miniconda@v2`?

Reply:
Conda is slow and not needed here. I would keep this.